deltacat/tests/compute/test_util_create_table_deltas_repo.py

```diff
@@ -251,6 +251,7 @@ def create_src_w_deltas_destination_rebase_w_deltas_strategy(
     ds.commit_partition(staged_partition, **ds_mock_kwargs)
 
     # get streams
+    # TODO: Add deltas to destination stream
```

Comment:
What is this TODO?

Reply:
In the future, we discussed making the destination stream non-empty to catch some issues with the test cases. I think it's fine if we keep the TODO to keep the task scope smaller? What do you think

Reply:
This mirrors the production behavior anyway.